hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -70,7 +69,8 @@ instance Tx tx => SignableRepresentation (Snapshot tx) where
   getSignableRepresentation = encodeUtf8 . show @Text
 
 data ClientResponse tx
-  = PeerConnected Host
+  = PeerConnected Party
+  | PeerDisconnected Party
```

Comment:
Would say `PartyConnected` and `PartyDisconnected` is more correct as this is really based on the hydra-node (with a configured `Party`) sending us pings?